src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,27 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <UserInfo user={user} />
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
+
+      {comments.length ? (
+        <CommentList comments={comments} />
+      ) : (
+        <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
I see that u have already used the `strong` tag for similar functionality.
Let's use it here too

Reply:
not fixed

Reply:
I think this is important for tests, because they fail if I change it